Pod/Classes/URBNAlertStyle.m

```diff
@@ -153,17 +153,21 @@ - (id)copyWithZone:(NSZone *)zone {
     // UIColors
     styler.buttonBackgroundColor = [[UIColor alloc] initWithCGColor:self.buttonBackgroundColor.CGColor];
     styler.destructionButtonBackgroundColor = [[UIColor alloc] initWithCGColor:self.destructionButtonBackgroundColor.CGColor];
+    styler.destructiveButtonTitleColor = [[UIColor alloc] initWithCGColor:self.destructiveButtonTitleColor.CGColor];
```

Comment:
If I do not do this, setting the global styles (button background color, title color, etc) get overwritten with the defaults. I am open to any other way of handling this